README.md

```diff
@@ -280,4 +280,4 @@ Post an issue here on GitHub if you have any problems, and create a pull request
 
 It seems like my preproc/{task}__{split}.data has nothing in it!
 
-This probably means that you probably ran the script before downloading the data for that task. Thus, delete the file from preproc and then run main.py again to build the data splits from scratch.
+This probably means that you probably ran the script before downloading the data for that task. Thus, delete the file from preproc and then run main.py again to build the data splits from scratch.
```

Comment:
Try to remove files from the PR when they're not really changing. Includes this file and others below.